pennylane_lightning/core/src/simulators/lightning_gpu/StateVectorCudaManaged.hpp

```diff
@@ -2128,6 +2149,38 @@ class StateVectorCudaManaged
                            stream_id);
     }
 
+    /**
+     * @brief Set values for a batch of elements of the state-vector. This
+     * method is implemented by the customized CUDA kernel defined in the
+     * DataBuffer class.
+     *
+     * @param num_indices Number of elements to be passed to the state vector.
+     * @param values Pointer to values to be set for the target elements.
+     * @param wires Wires of the target elements.
+     * @param is_left_significant If true, the most significant wire is on the
```

Comment:
I'm not sure if I'm getting what `is_left_significant` is defining here. Could you please expand on that?